### ISSUE TITLE
An idea, expand right first in Clojure mode

### ISSUE THREAD

**Issue by arichiardi** [magnars/expand-region.el#172]
I hope it is the right place for suggesting this.
In Clojure, we usually have ```let``` block like this (where | is point):

```clojure
(let [binding1 |(map f coll)
       binding2 :constant]
  ..do..)
```

I always want, but maybe it is just my brain, to select ```binding1 |(map f coll)``` together so that I can go to my repl and easily paste it after a def, like: ```(def binding1 (map f coll))```, but at the second ```expand-region``` actually selects the whole content of the ```let```.

My question is, is there a way to say: ok, I want an additional step that sees that I am in a ```let``` or sees that something is on my right, and selects that before expanding to the whole thing?

I hope it's clear!
Thanks for this Magnars and contributors :+1:

**Comment by magnars** [comment]
Good idea! I like it. If you haven't heard from me on this in a week, would you mind giving me another nudge here?

**Comment by arichiardi** [comment]
Sure! Thanks a lot! :bow:
